docs/using/security-threat-tool.md

```diff
@@ -2,30 +2,33 @@
 
 The Security Threat Tool runs regular checks against connected databases, alerting you if any servers pose a potential security threat.
 
-All checks run on the PMM Client side. Results are sent to PMM Server where a summary count is shown on the *Home Dashboard*, with details in the *PMM Database Checks* dashboard.
+### Anonymous and registered checks
+All checks are hosted on Percona Platform. PMM Server automatically downloads them from here when the Security Threat Tool is enabled in PMM. 
 
-Checks are automatically downloaded from Percona Platform and run every 24 hours. (This period is not configurable.)
+By default, PMM has access to a set of anonymous checks, which can be downloaded even if PMM is not connected to Percona Platform. 
+As soon as you connect your PMM instance to Percona Platform, you get additional access to registered checks, which offer more advanced database health information.
 
-**Check results data *always* remains on the PMM Server.** It is not related to anonymous data sent for Telemetry purposes.
+​To see the complete list of available checks, see the [Security Checks for PMM](https://docs.percona.com/percona-platform/checks.html) topic in the Percona Platform documentation.  
 
-The *Failed security checks* panel on the *Home Dashboard* shows the number of failed checks classed as *critical* (red), *major* (amber), and *trivial* (blue).
 
-![!Failed security checks panel](../_images/PMM_Home_Dashboard_Panels_Failed_Security_Checks.jpg)
+## How to enable
 
-!!! note alert alert-light "Key"
-    <b style="color:#e02f44;">Critical</b> (Red) / <b style="color:#e36526;">Major</b> (Amber) / <b style="color:#5794f2;">Trivial</b> (Blue)
+By default, the Security Threat Tool (STT) is disabled. To enable it, select <i class="uil uil-cog"></i> *Configuration* → <i class="uil uil-setting"></i> *Settings* → *Advanced Settings*. ([Read more](../how-to/configure.md#advanced-settings)).
 
-Details are in the *PMM Database Checks* dashboard (select *PMM* → *PMM Database Checks*).
+Enabling STT in the settings also causes the PMM server to download STT checks from Percona Platform and run them once. This operation runs in the background so even though the settings update finishes instantly, it might take some time for the checks to complete download and execution and the results (if any) to be visible in the *PMM Database Checks* dashboard.
```

Comment:
We could change as follows:
Enabling STT in the settings causes the PMM server to download STT checks from the Percona Platform and run them once. This operation runs in the background, so even though the settings update finishes instantly, it might take some time for the checks to complete download and execution and the results (if any) to be visible in the *PMM Database Checks* dashboard.